app/views/books/edit.html.erb

```diff
@@ -1,6 +1,15 @@
-<h1>Editing Book</h1>
+<div style="margin-top:50px;" class="container-fluid col-md-8 col-md-offset-2">
+  <div class="panel panel-primary">
 
-<%= render 'form', book: @book %>
+    <div class="panel-heading">
+      <div class="panel-title">Edit book</div>
```

Comment:
BTW, it's generally considered good HTML style to use semantic tags like `h1` where applicable, rather than making everything a `div`. It makes the HTML a bit more readable, it allows for standard styles, it's important for accessibility (e.g., screen readers for blind users), and it's important for SEO (the h1 tag indicates the most important keywords on the page, for instance). So try to make good use of tags like h1, h2, p, etc.; and only use div for things that are basically just containers for layout.